indexing-service/src/main/java/io/druid/indexing/worker/executor/ExecutorLifecycle.java

```diff
@@ -91,30 +111,6 @@ public void start()
       throw Throwables.propagate(e);
     }
 
-    // Spawn monitor thread to keep a watch on parent's stdin
```

Comment:
would it be ok to make it a config in runtime.props, also a flag while starting up the middlemanager indicating if we want to start with a clean state by killing any left over tasks from previous run can be useful.